Kconfig

```diff
@@ -104,4 +104,17 @@ config DOMD_DTB_PATH
 config TA_DEPLOY_DIR
 	string "Location of optee TA's deploy dir"
 
+config AOS_ENABLE_DOMU
```

Comment:
Not directly related to AOS. `AOS_` prefix can be removed.

Reply:
This config will be exported globally over the project it should have some specific naming to prevent collision. Since now it affect only AoS, I used such naming. Can you suggest anything else?

Reply:
`DOMU_..` is good prefix in this case: `DOMU_ENABLE`. Looks good for me.